product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Connecting from AWS 
+
+---
+
+Two different methods enable you to connect to your private cluster from your application's VPC in AWS. Each method offers different levels of accessibility and security. The VPC endpoint method is recommended and is most commonly used. However, you can also use the VPC peering connection method if required by your organization.
+
+- VPC endpoint method
+- VPC peering connection methodf
+
+## AWS VPC endpoint (recommended)
+
+ AWS PrivateLink service is a network interface that securely connects a private IP address from your AWS VPC to an external service. You grant access only to a single cluster instead of the entire BigAnimal resource VPC, thus ensuring maximum network isolation. For more information, see [VPC endpoint services (AWS PrivateLink)](https://docs.aws.amazon.com/vpc/latest/privatelink/endpoint-service.html).
```

Comment:
The switch from "VPC endpoint" to "AWS PrivateLink" is a bit jarring - I think we should probably explain in a bit more depth how the two things are related.

Reply:
:-) Updated documentation to indicate they are one and the same.